lightning/src/ln/outbound_payment.rs

```diff
@@ -915,7 +915,6 @@ impl OutboundPayments {
 			return Err(PaymentSendFailure::PathParameterError(path_errs));
 		}
 		if let Some(amt_msat) = recv_value_msat {
-			debug_assert!(amt_msat >= total_value);
```

Comment:
Hopefully this won't disrupt other testing much, I couldn't find any other way to send an MPP while being able to set the `total_msat` to be lower for testing, and all of the send payment methods seem to call `pay_route_internal` anyway so I figured I might not be able to avoid this.